src/components/Aside/Aside.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { BrowserRouter as Router, NavLink } from 'react-router-dom';
+
+class Aside extends React.Component {
+  render() {
+    return (
+      <Router>
+        <aside>
+          <ul className="p-list">
+            <li><NavLink exact to="/buttons" activeClassName="active">Buttons</NavLink></li>
+            <li><NavLink exact to="/cards" activeClassName="active">Cards</NavLink></li>
+            <li><NavLink exact to="/switches" activeClassName="active">Switches</NavLink></li>
+          </ul>
```

Comment:
Good start but might be worth passing an array in the future and iterating over it to create the navigation.

Reply:
Yep, that's the plan once I figure out how to do it 😅